docs/architecture.md

```
@@ -0,0 +1,271 @@
+- x
+
+## functions
+
+- x
+
+## view
+
+- <div>
+   <div>
+      <div></div>
+      <img />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)